src/components/TagSelector/index.jsx

```diff
@@ -45,10 +45,10 @@ class TagSelector extends Component {
       backgroundColor: tag.backgroundColor
     }));
     const menuValues = value.map((tag) => ({
```

Comment:
Question: Is the choice to use `tag` as the name of the element to map throughout this PR intentional? I see we're doing `tag.tag` throughout and wonder if renaming to `contactTag` or similar would be helpful

Reply:
Yeah. Intentional. It can be renamed to `contactTag` or `campaignContactTag`